live_view/live_view_studio/lib/live_view_studio_web/live/light_live.ex

```diff
@@ -0,0 +1,50 @@
+defmodule LiveViewStudioWeb.LightLive do
+  use LiveViewStudioWeb, :live_view
+  def mount(_params, _session, socket) do
```

Comment:
@samu975 how the mount works?

Reply:
mount is for assigning the initial of the live view process. Takes three parameters: params(a map of the current query params), session(a map with the private session data), and socket (a struct representing the WebSocket connection)